actions/src/issues/checkIssueLabels.js

```diff
@@ -30,7 +30,7 @@ const checkLabels = async () => {
   const user = context.payload.sender.login;
 
   if (
-      label.name === GOOD_FIRST_LABEL &&
+    label.name === GOOD_FIRST_LABEL &&
       !whitelist.goodFirstIssue.includes(user)) {
```

Comment:
I would do something like
```suggestion
    label.name === GOOD_FIRST_LABEL &&
    !whitelist.goodFirstIssue.includes(user)
  ) {
```

Reply:
Done, 
I forget to do the file reformat thing. Thanks for reminding.